plugin/metrics/prometheus/metricsstore/reader_test.go

```diff
@@ -61,22 +64,36 @@ var defaultConfig = config.Configuration{
 	LatencyUnit:                 "ms",
 }
 
+func createTracingProvider() trace.TracerProvider {
+	exporter, err := stdout.New(stdout.WithPrettyPrint())
+	if err != nil {
+		panic(err)
```

Comment:
you won't need it because of above change, but if you did, the correct way to do this is to pass `t` into the function and do `require.NoError(t, err)`

Reply:
Thanks much. Do I also need to change the name of the function? like `TestInitTracingProvider`.

Reply:
no, what you have is fine. You could make it shorter `tracerProvider(t)`. You certainly don't want to name it TestXxx, as that would make it an actual unit test, while it's just a helper function.